task/buildah-remote/0.1/buildah-remote.yaml

```diff
@@ -104,7 +104,7 @@ spec:
   - env:
     - name: COMMIT_SHA
       value: $(params.COMMIT_SHA)
-    image: quay.io/redhat-user-workloads/rhtap-build-tenant/multi-arch-controller/hacktask-image-multi-platform-controller:build-dfd73-1698382902@sha256:d733531a07ede2e73500c500a33cdaf78332db146ffdc23d3c1a71b253a734b6
+    image: quay.io/redhat-appstudio/multi-platform-runner:01c7670e81d5120347cf0ad13372742489985e5f
```

Comment:
IIUC the commit-hash tag for this image is pinned in the taskgen script, which means the image doesn't self-update (via Renovate). Updating the image would require bumping the tag in https://github.com/redhat-appstudio/multi-platform-controller/blob/main/cmd/taskgen/main.go, and bumping the taskgen image tag in `hack/generate-buildah-remote.sh`

Could be improved in the future?

Reply:
It definitely can be improved.